pkgs/top-level/all-packages.nix

```diff
@@ -20158,7 +20158,7 @@ in
 
   joe = callPackage ../applications/editors/joe { };
 
-  josm = callPackage ../applications/misc/josm { };
+  josm = callPackage ../applications/misc/josm { jre = jdk11; };
```

Comment:
Why does it use actually `jdk` instead of `jre`?

Reply:
Since we have no `jre11` alias or `jdk11.jre`, only `jre = jre8`.

Reply:
There are 3 other packages in `all-packages.nix` that use `jre = openjdk11` as well.